Sources/Purchasing/StoreKit1/ProductsFetcherSK1.swift

```diff
@@ -188,6 +188,12 @@ extension ProductsFetcherSK1: SKProductsRequestDelegate {
         }
     }
 
+    func clearCache() {
+        queue.async {
```

Comment:
Explicit `self` per the style guide:
```suggestion
        self.queue.async {
```